app/konnect/configure/runtime-manager/runtime-groups/declarative-config.md

```diff
@@ -85,31 +109,27 @@ For this example, let's add a new service.
 the example with your generated UUID:
 
     ```yaml
```

Comment:
Filename should be `kong.yaml`, while in L108 we are referring to `konnect.yaml`